packages/google_maps_flutter/google_maps_flutter/ios/Classes/JsonConversions.h

```diff
@@ -5,15 +5,28 @@
 #import <Flutter/Flutter.h>
 #import <GoogleMaps/GoogleMaps.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
 @interface FLTGoogleMapJsonConversions : NSObject
-+ (bool)toBool:(NSNumber *)data;
-+ (int)toInt:(NSNumber *)data;
-+ (double)toDouble:(NSNumber *)data;
-+ (float)toFloat:(NSNumber *)data;
-+ (CLLocationCoordinate2D)toLocation:(NSArray *)data;
-+ (CGPoint)toPoint:(NSArray *)data;
-+ (NSArray *)positionToJson:(CLLocationCoordinate2D)position;
-+ (UIColor *)toColor:(NSNumber *)data;
-+ (NSArray<CLLocation *> *)toPoints:(NSArray *)data;
-+ (NSArray<NSArray<CLLocation *> *> *)toHoles:(NSArray *)data;
++ (bool)getBoolFromData:(NSNumber *)data;
++ (int)getIntFromData:(NSNumber *)data;
++ (double)getDoubleFromData:(NSNumber *)data;
++ (float)getFloatFromData:(NSNumber *)data;
++ (CLLocationCoordinate2D)getLocationFromData:(NSArray *)data;
++ (CGPoint)getPointFromArray:(NSArray *)data;
++ (NSArray *)getJsonFromLocation:(CLLocationCoordinate2D)location;
++ (UIColor *)getColorFromData:(NSNumber *)data;
++ (NSArray<CLLocation *> *)getPointsFromData:(NSArray *)data;
++ (NSArray<NSArray<CLLocation *> *> *)getHolesFromData:(NSArray *)data;
++ (nullable NSDictionary<NSString *, id> *)getJsonFromPosition:
+    (nullable GMSCameraPosition *)position;
++ (NSDictionary<NSString *, NSNumber *> *)getJsonFromPoint:(CGPoint)point;
++ (nullable NSDictionary *)getJsonFromCoordinateBounds:(nullable GMSCoordinateBounds *)bounds;
++ (nullable GMSCameraPosition *)getOptionalCameraPostionFromData:(nullable NSDictionary *)data;
```

Comment:
I've never seen nullability encoded in a name like this. Is this version used enough to need a wrapper, vs callers doing the short-circuit on null (which is more common for a nullable language than having two versions of the API)?